apps/antalmanac/src/routes/Error.tsx

```diff
@@ -0,0 +1,25 @@
+import { Typography, Button, Stack, Box } from '@mui/material';
+import { Link } from 'react-router-dom';
+
+const ErrorPage = () => (
+    <Box sx={{ padding: '1rem' }}>
+        <Typography variant="h3" component="h1">
+            Oops! Something went wrong 🥺 👉👈
+        </Typography>
+        <Stack spacing={2} sx={{ paddingY: '1rem' }}>
+            <Typography variant="h5" component="p">
+                This error may be caused by you having an out of date version of AntAlmanac.
```

Comment:
```suggestion
                This error may be caused by your browser having an out of date version of AntAlmanac.
```
nit: "you" may feel too direct for users